doc/rados/operations/health-checks.rst

```diff
@@ -1048,7 +1048,10 @@ PG_NOT_SCRUBBED
 _______________
 
 One or more PGs has not been scrubbed recently.  PGs are normally scrubbed
-within every ``osd_scrub_max_interval``, and this warning triggers when
+within every configured interval which is specified
+:ref:`osd_scrub_max_interval <osd_scrub_max_interval>` globally. And this
+interval can be overriden on per-pool basis with
+:ref:`scrub_max_interval <scrub_max_interval>`. And this warning triggers when
```

Comment:
Same as above but maybe got with the opposite of what you choose in the first instance.